### ISSUE TITLE
Safe status shows panic runtime error

### ISSUE THREAD

**Issue by Aracki** [starkandwayne/safe#192]
All of the commands I have tries worked, but `safe status` is throwing:

```panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x12a3b57]

goroutine 1 [running]:
github.com/starkandwayne/safe/vendor/github.com/cloudfoundry-community/vaultkv.(*Client).Health(0xc0004bd740, 0x1, 0x0, 0x0)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/vendor/github.com/cloudfoundry-community/vaultkv/sys.go:210 +0x117
github.com/starkandwayne/safe/vault.(*Vault).Sealed(0xc00008a008, 0x0, 0x0, 0x0)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/vault/seal.go:47 +0x3a
main.main.func7(0x1398f8a, 0x6, 0x17ba488, 0x0, 0x0, 0xc0000afe01, 0x102a34f)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/main.go:678 +0x13e
main.(*Runner).Execute(0xc00004ef10, 0x1398f8a, 0x6, 0x17ba488, 0x0, 0x0, 0x0, 0x0)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/runner.go:117 +0x181
main.main()
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/main.go:3948 +0x3098
```

Safe version: `1.4.0`
Vault version: `0.10.1`

**Comment by thomasmitchell** [comment]
Oh, darn. Fixed this a month ago but never pulled in the new commit.

Seems like this occurs if the request to `/sys/health` would have erred for a different reason. At the time, it was trying to unmarshal the JSON from the response body without checking for a transport error.

Anyways, it'll be in 1.4.1. Should be fixed by 91cdea68b44eda184ea3575313293f78e2e06a63

**Comment by thomasmitchell** [comment]
Released in 1.4.1. Feel free to reopen if you still have this issue.

**Comment by Aracki** [comment]
Thanks for explanation 👍
